frontend/sozisel-app/src/components/TemplatesList/TemplateList.tsx

```diff
@@ -15,6 +15,7 @@ import { ReactElement } from "react";
 import SearchBar from "./SearchBar/SearchBar";
 import Snackbar from "@material-ui/core/Snackbar";
 import TemplateCard from "./TemplateCard/TemplateCard";
+import { useCallback } from "react";
```

Comment:
This should be grouped together with `useState` Are you sure that you have a working linter? The linter would pass that code.